Adding test for increasing the coverage for speechtotext component

subhahu123 opened the pull request:
<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->
Fixes #1467

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link
<!-- Replace "PR_NUMBER" with your pull request number. This link is generated when your PR passes the travis tests.A sample link can look like https://pr-200-fossasia-susper.surge.sh -->

#### Changes proposed in this pull request:

<!-- Changes: Add here what changes were made in this issue and if possible provide links. -->

<!-- Demo Link: Add here the link where you changes can be seen. -->

- https://pr-1458-fossasia-susper.surge.sh

<!-- Screenshots for the change: Add here the screenshot of the fix. -->

simsausaurabh commented on src/app/speechtotext/speechtotext.component.spec.ts:
```diff
@@ -1,14 +1,30 @@
   let fixture: ComponentFixture<SpeechtotextComponent>;
+  let message: any;
+  let ticks: any;
+  let timer: any;
```
Don't just define type as ```any```, always try to declare exact type of variable. And please also fix it at all instances.

subhahu123 replied:
any is used in component.ts file i m just keeping it sync with it
`

simsausaurabh commented on src/app/speechtotext/speechtotext.component.spec.ts:
```diff
@@ -31,4 +47,89 @@ describe('SpeechtotextComponent', () => {
+
+  it('should create div with id spch', () => {
+    let compiled = fixture.debugElement.nativeElement;
+    expect(compiled.querySelector('div#spch')).toBeTruthy();
```
You should write tests which tests something meaningful.

subhahu123 replied:
removed

simsausaurabh reviewed (changes requested):
You also need to squash your commits, and fill the PR description properly.

subhahu123 committed:
Adding test for increasing the coverage
Update speechtotext.component.spec.ts
Update speechtotext.component.spec.ts
Update speechtotext.component.spec.ts
Update speechtotext.component.spec.ts
Update speechtotext.component.spec.ts
Update speechtotext.component.spec.ts
Update speechtotext.component.spec.ts
Update speechtotext.component.spec.ts
Update speechtotext.component.spec.ts

praveenojha33 reviewed (approved):
LGTM :+1:
